src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNet/AspNetCoreMvc2Integration.cs

```diff
@@ -169,6 +169,12 @@ public AspNetCoreMvc2Integration(object actionDescriptor, object httpContext)
                        .Start(Assembly.GetCallingAssembly(), mdToken, opCode, nameof(BeforeAction))
                        .WithConcreteTypeName(DiagnosticSource)
                        .WithParameters(diagnosticSource, actionDescriptor, httpContext, routeData)
+                       .WithNamespaceAndNameFilters(
+                            ClrNames.Void,
+                            ClrNames.Ignore,
```

Comment:
Why do we ignore the first parameter's type in all these methods? It's `"Microsoft.AspNetCore.Mvc.Internal.MvcCoreDiagnosticSourceExtensions"` (or the constant `DiagnosticSource`).

Reply:
I'm not sure if this is correct or not, and I will need to re-verify: I think I recall that for the version range we support, the names/namespaces changed.
I *DEFINITELY* remember thinking that the parameter had no consequence on what WE were doing, so I had decided it was okay at the time.
With the push toward being more explicit, maybe it would be good to specify them.

Reply:
I opened a follow up for this